### PR TITLE
Handle multiple opening times in the same day

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/iss/ServiceOpening.js [hunk 1]
```diff
@@ -73,13 +73,15 @@ export default class ServiceOpening {
                 end: close,
             }, openingHours);
 
-            openingTimes[open.diff(now(), "days")] = serviceOpeningHours;
-            closingTimes[close.diff(now(), "days")] = serviceOpeningHours;
+            openingTimes.push(serviceOpeningHours)
```
Comment (kane-c) [inline src/iss/ServiceOpening.js]: Semicolons

File: src/components/CollapsedOpeningTimes.js [hunk 2]
```diff
@@ -74,7 +87,10 @@ export default class CollapsedOpeningTimes extends React.Component {
         ];
 
         let openingHours = _(this.props.object.openingTimes)
-            .sortBy(record => order.indexOf(record.day));
+            .sortBy(record =>
+                (order.indexOf(record.day) * 24) +
+                (parseInt(record.open))
```
Comment (kane-c) [inline src/components/CollapsedOpeningTimes.js:92]: What is `record.open`? Is it like `"01:00:00"`? If so, `parseInt` would always just give `1`

Comment (DanielHeath) [inline src/components/CollapsedOpeningTimes.js:92]: `parseInt("12:30:00")` gives `12`

Comment (DanielHeath) [inline src/components/CollapsedOpeningTimes.js:92]: The idea is that if you're open more than once in a day, we want the opening times to show the morning entry before the afternoon one..

Comment (kane-c) [inline src/components/CollapsedOpeningTimes.js:92]: Huh strange. I tried it in the console and it was always 1 but now that I try more it's different. OK, ignore me.

**Comment by DanielHeath** [comment]
@kane-c ++?

**Comment by kane-c** [comment]
++

**Pull request merged**